ACKategoriesCore/RandomExtensions/String+Random.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+extension String: Randomizable {
+    public static func random() -> String {
+        String.random(minLength: 5)
+    }
+
+    /// - Parameters:
+    ///     - minLength: Minimum length of a returned String
+    ///     - maxLength: Maxium length of a returned String
+    ///     - allowedLetters: Letters that will be used in the randomization
+    /// - Returns: Randomized String
+    public static func random(minLength: Int = 5,
+                              maxLength: Int = 10,
+                              allowedLetters: String = "abcdefghijklmnopqrstuvwxyzěščřžýáíéABCDEFGHIJKLMNOPQRSTUVWXYZĚŠČŘŽÝÁÍÉ0123456789") -> String {
```

Comment:
It says `allowedLetters`, but I can also see digits in there. Maybe `allowedCharacters` would be better?

Reply:
Sure, good idea 👍